src/bundle/Resources/public/scss/_modals.scss

```diff
@@ -21,6 +21,19 @@
 
     .modal-body {
         @include modal-body();
+
+        .ibexa-modal__option-label {
+            margin-bottom: 0;
+            margin-top: 1rem;
```

Comment:
We do not use rem unit directly, please use `calculateRem()`

Reply:
Resolved in https://github.com/ibexa/admin-ui/pull/1321/commits/3613e0030e063189d153e256c197edf2e90e778f